modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase

```diff
@@ -314,6 +314,109 @@ definition {
 			value1 = "Amortiguador");
 	}
 
+	@description = "COMMERCE-5802. Use a filter always check that the results are correct"
+	@ignore = "true"
+	@priority = "3"
+	test UseFilters {
+		property portal.acceptance = "false";
+
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
+
+		Click(locator1 = "CommerceEntry#FILTER_DROPDOWN_MENU");
+
+		Click(
+			key_categoryFilter = "Category",
+			locator1 = "CommerceEntry#FILTER_SELECT_CATEGORY");
+
+		Check.checkNotVisible(
+			checkboxName = "Brake System",
+			locator1 = "Checkbox#ANY_CHECKBOX");
+
+		Click(
+			key_menuItem = "Add Filter",
+			locator1 = "CommerceEntry#ANY_MENU_ITEM");
+
+		Click(locator1 = "CommerceEntry#FILTER_DROPDOWN_MENU");
+
+		AssertElementPresent(
+			key_entryName = "Premium Brake Fluid",
+			locator1 = "CommerceEntry#TABLE_LIST_TITLE");
```

Comment:
https://github.com/liferay-commerce/liferay-portal/pull/857/files?file-filters%5B%5D=.testcase#r606166979

@pedro-michel As per the previous request, to test that the filter works correctly we should assert that an element that does not belong to the selected category is not present, instead of checking that one we know should be present actually is. 

Thx.

Reply:
In progress

Reply:
Done